pkg/server/handler.go

```diff
@@ -130,7 +129,8 @@ func WithClusterScope(apiHandler http.Handler) http.HandlerFunc {
 		case clusterName == logicalcluster.Wildcard:
 			// HACK: just a workaround for testing
 			cluster.Wildcard = true
-			fallthrough
+			//fallthrough
```

Comment:
no fallthrough anymore?

Reply:
I took this out because "wildcard, but cluster.Name is actually system:admin" was breaking things in the CRD lister. All the e2es continue to pass... 🤷 😄